openlibrary/templates/books/edit.html

```diff
@@ -120,6 +121,23 @@ <h3 class="editFormBookAuthors">
                         $:render_template("books/edit/web", work, prefix="work--")
                     </div>
                 </fieldset>
+                <fieldset class="major" id="identifiers">
+                    <legend>$_("Work Identifiers")</legend>
+                    <div class="formBack">
+                        <div id="id-errors" class="note" style="display:none"></div>
+                        <div class="formElement">
+                            <div class="label">
+                                <label for="select-id">$_("Do you know any identifiers for this work?")</label>
+                                <span class="tip">$_("Like, Wikidata?")</span>
+                            </div>
+                        </div>
+                    </div>
+                    <div id="hiddenIdentifiers"></div>
+                    <div id="identifiers-display-works">
+                        $ admin = str(ctx.user.is_admin() or ctx.user.is_super_librarian())
+                        $:render_component('IdentifiersInput', attrs=dict(assigned_ids_string=work.get_identifiers().values(), output_selector='#hiddenIdentifiers', id_config_string=work_config, input_prefix='work--identifiers', admin=admin))
```

Comment:
This is actually on the same html page as the edition identifiers, so we'll need to give this a unique id!

```suggestion
                    <div id="hiddenWorkIdentifiers"></div>
                    <div id="identifiers-display-works">
                        $ admin = str(ctx.user.is_admin() or ctx.user.is_super_librarian())
                        $:render_component('IdentifiersInput', attrs=dict(assigned_ids_string=work.get_identifiers().values(), output_selector='#hiddenWorkIdentifiers', id_config_string=work_config, input_prefix='work--identifiers', admin=admin))
```